packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts

```diff
@@ -782,43 +789,39 @@ export const InstanceProperties = {
   description: define(null, ["es.symbol", "es.symbol.description"]),
   dotAll: define(null, ["es.regexp.dot-all"]),
   drop: define(null, [
+    "es.iterator.drop",
+    ...IteratorDependencies,
     "esnext.async-iterator.drop",
     ...AsyncIteratorDependencies,
-    "esnext.iterator.drop",
-    ...IteratorDependencies,
-  ]),
-  emplace: define("instance/emplace", [
-    "esnext.map.emplace",
-    "esnext.weak-map.emplace",
```

Comment:
With the current approach, it should be injected on constructors call, and that's already happening. Here are missed dependencies and in this way anyway it could cause a error. However, in case of pure mode + targets, it could fail because of https://github.com/babel/babel-polyfills/issues/166.